scipy/stats/_continuous_distns.py

```diff
@@ -9124,76 +9123,81 @@ def _rvs(self, chi, size=None, random_state=None):
         return out
 
     def _rvs_scalar(self, chi, numsamples=None, random_state=None):
-        # if chi <= 2.611:
+        # if chi <= 1.8:
         # use rejection method, see Devroye:
         # Non-Uniform Random Variate Generation, 1986, section II.3.2.
         # write: self.pdf = c * g(x) * h(x), where
         # h is [0,1]-valued and g is a density
-        # g(x) = d1 * chi**2 * x * exp(-chi**2 * (1 - x**2) / 2), 0 <= x <= 1
+        # we use two ways to write f
+        #
+        # Case 1:
+        # note that the pdf of ARGUS converges to 3*x*sqrt(1-x**2)
```

Comment:
```suggestion
```
This note suggested to me that the sampling method relied on a small chi approximation. After careful review, I saw that this fact is not used in the method, so I think it can be removed from here.  If you want the fact to be emphasized, perhaps add a more direct unit test:

```
import numpy as np
from scipy.stats import argus

np.random.seed(0)
chi = 1e-9
dist = argus(chi)
rvs = dist.rvs(100)

np.random.seed(0)
np.random.uniform(size=100) # burn the samples for U
# alternatively, swap sampling of U and V in _rvs_scalar
V = np.random.uniform(size=100)
rvs2 = np.sqrt(1 - V**(2/3))

np.testing.assert_allclose(rvs, rvs2)
```